src/main/java/gmail/smoljarn/lesson38_generic/FirstNumbers.java

```diff
@@ -0,0 +1,18 @@
+package gmail.smoljarn.lesson38_generic;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class FirstNumbers {
+    public static List<Integer> getFirstNumbers(List<Integer> numbers) {
+
+        List<Integer> result = new ArrayList<>();
+
+        for (Integer number : numbers) {
+            if (!result.contains(number)) {
+                result.add(number);
+            }
+        }
+        return result;
+    }
```

Comment:
Дуже добре!!!! Respect 👍🏻 
```suggestion
    public static List<Integer> getUniqueNumbers(List<Integer> numbers) {

        List<Integer> result = new ArrayList<>();

        for (int number : numbers) {
            if (!result.contains(number)) {
                result.add(number);
            }
        }
        return result;
    }
```